src/interfaces/IREP15.sol

```diff
@@ -0,0 +1,153 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.26;
```

Comment:
does not need to specifiy static solc version.

Reply:
Why? I have seen OZ use it.

Reply:
if your logic does not include special features from latest solc version 0.8.26, I think it is reasonable to reduce constraints on solc version since many dapps or 3rd parties might use our interfaces in their project and their contract version might not be latest.

Reply:
okay

Reply:
Noted that as named parameter in mapping type only supported in 0.8.18 and later.